lib/runAll.js

```diff
@@ -128,61 +160,86 @@ export const runAll = async (
   // Set of all staged files that matched a task glob. Values in a set are unique.
   const matchedFiles = new Set()
 
-  for (const [index, files] of stagedFileChunks.entries()) {
-    const chunkTasks = generateTasks({ config, cwd, gitDir, files, relative })
-    const chunkListrTasks = []
+  for (const [configDir, files] of Object.entries(configDir2Files)) {
```

Comment:
Are these sorted in the way that a deeper config file always takes precedence? I think that would make sense.

This way we could always load the "root" config with this new method, and then apply more nested config files over that.

Reply:
> Are these sorted in the way that a deeper config file always takes precedence? I think that would make sense

Yes it is.